airflow/opt/providers/etna/etna/etls/metis_linker.py

```diff
@@ -93,29 +93,35 @@ def data_frame_update(self, model_name, script, tail, update, metis, model):
                 if script['format']=="csv":
                     separator = ","
                 if script['format']=="tsv":
-                    separator = "\\t"
+                    separator = "\t"
                 if script['format']=="auto-detect":
                     if re.search("csv$", file.file_path) is not None:
                         separator = ","
                     elif re.search("tsv$", file.file_path) is not None:
-                        separator = "\\t"
+                        separator = "\t"
```

Comment:
Encountered a couple other errors during testing:
- For whatever reason, this "\\t" separator mostly works but can cause the rightmost data column to fill as NaN even with `na_filter=False`.  I'm not sure why I'd had "\\t" rather than "\t" anymore, so since "\t" seems to work better, switching to that here and for the analogous vulcan code too.